src/main.js

```diff
@@ -412,12 +435,12 @@ function isWindows() {return (process.platform === 'win32');}
 // ====================================================================================================================
 
 // Returns the execution path of this application.
-ipcMain.handle('get-appPath', () => {
+ipcMain.handle('get-appPath', (event) => {
```

Comment:
event is unused and can be omitted from here

Reply:
Done.  Removed all the unused event parameters here.